lib/circuit.js

```diff
@@ -157,13 +157,7 @@ class CircuitBreaker extends EventEmitter {
 
     // The user can pass in a Status object to inialize the Status/stats
     if (this.options.status) {
-      // Do a check that this is a Status Object,
-      if (this.options.status instanceof Status) {
-        this[STATUS] = this.options.status;
-      } else {
-        // should it error if it is not a Status Object
-        throw new TypeError('Not a Status Object.  The status option must be an instance of Status');
-      }
+      this[STATUS] = new Status(this.options.status);
```

Comment:
Shouldn't this be like this?
```suggestion
      this[STATUS] = new Status({ stats: this.options.status });
```
https://github.com/nodeshift/opossum/blob/16354f2a7b1f7f30f62acc2481f6b0f2e9879cef/lib/status.js#L90-L92

Reply:
derp,  yup